skbio/util/_warning.py

```diff
@@ -20,3 +20,14 @@ class EfficiencyWarning(Warning):
 
     """
     pass
+
+
+class RepresentationWarning(Warning):
+    """Warn about substitions, assumptions, or particular alterations
```

Comment:
substitions -> substitutions

The first sentence of the docstring needs to be on a single line that is <= 79 characters, otherwise it gets truncated.